gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -2309,7 +2326,7 @@ subroutine Initialize ( gc, import, dumexport, clock, rc )
           end do
 
 !       endif
-    enddo
+       enddo
```

Comment:
```suggestion
     end do
```
Is the indentation off?

Reply:
With emacs you can select a region spanning the entire file and then use `ESC-x f90-indent-region`  Some chance of failure if the file does something too exotic with the preprocessor, but generally works.

Reply:
Ah - I'd not noticed it was in the "big" file - thought it was in your new module.   Definitely don't want to do it in this PR.  (Better to have it separate.)  But at least you have that tool in your editor arsenal now.

Reply:
And ... there is always a chance that your emacs and my emacs have different settings for indents.   I think I'm using the defaults, but have worked with NASA codes that required customization in the past.    I've had SE's say that it should be a constant level of indent, but Emacs does 2 chars for some things and 3 for others.   I like that style, but I can see it as being tedious for someone that is not working with Emacs.